client/product-details/index.js

```diff
@@ -7,27 +7,39 @@
 import { initializeBnplSiteMessaging } from './bnpl-site-messaging';
 
 jQuery( function ( $ ) {
+	// Check for required global variables
```

Comment:
Good one, can we expand a bit what's contained in this variable for readers?

Reply:
Done here https://github.com/Automattic/woocommerce-payments/pull/7153/commits/3ad3c2aaf61dcddb961168d7eadfeae460dd7ac9